packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -53,18 +51,16 @@ function SelectionTool(props: Props) {
   });
 
   const computeSelection = useCallback(
-    (evt: CanvasEvent<PointerEvent>) => {
+    (evt: CanvasEvent<PointerEvent>): Selection => {
       assertDefined(startEvt);
-      const { dataPt: startPoint, worldPt: worldStartPoint } = startEvt;
+      const { dataPt: dataStart, worldPt: worldStart } = startEvt;
 
-      const { worldPt: worldEndPt } = evt;
-      const boundWorldEndPoint = boundWorldPointToFOV(worldEndPt, camera);
+      const { worldPt: worldEnd } = evt;
+      const boundWorldEnd = boundWorldPointToFOV(worldEnd, camera);
 
       return {
-        startPoint,
-        endPoint: worldToData(boundWorldEndPoint),
-        worldStartPoint,
-        worldEndPoint: boundWorldEndPoint,
+        world: [worldStart, boundWorldEnd],
+        data: [dataStart, worldToData(boundWorldEnd)],
```

Comment:
I'm still hoping to be able to store the selection internally only in world space, and to call `onSelectionChange` and the `children` render prop with this world selection only. I feel like that's all you need during the interaction and that `onSelectionEnd` should be the only callback to receive the selection in data space (since that's when the selection becomes relevant to the app and may need to be persisted).

I'll keep working towards this.